test_scripts/API/SendLocation/009_SendLocation_boundary.lua

```diff
@@ -0,0 +1,160 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/24
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/TRS/embedded_navi/SendLocation_TRS.md
+-- Item: Use Case 1: Main Flow (Exception 1: wrong json)
+--
+-- Requirement summary:
+-- App requests SendLocation without deliveryMode and with other allowed and valid parameters
+--
+-- Description:
+-- App sends SendLocation with invalid payload (invalid json)
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered on SDL
+
+-- Steps:
+-- App sends SendLocation with invalid payload (invalid json)
+
+-- Expected:
+
+-- SDL validates parameters of the request
+-- SDL responds INVALID_DATA, success: false
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSendLocation = require('test_scripts/API/SendLocation/commonSendLocation')
+
+--[[ Local Variables ]]
+local lowerBoundRequest = {
+    longitudeDegrees = -180.0,
+    latitudeDegrees = -90.0,
+    address = {
+        countryName = "a",
+        countryCode = "a",
+        postalCode = "a",
+        administrativeArea = "a",
+        subAdministrativeArea = "s",
+        locality = "a",
+        subLocality = "s",
+        thoroughfare = "s",
+        subThoroughfare = "s"
+    },
+    timestamp = {
+        second = 0,
+        minute = 0,
+        hour = 0,
+        day = 1,
+        month = 1,
+        year = 0,
+        tz_hour = -12,
+        tz_minute = 0
+    },
+    locationName ="a",
+    locationDescription ="a",
+    addressLines = {"a"}, 
+    phoneNumber ="1",
+    deliveryMode = "PROMPT",
+    locationImage = 
+    { 
+        value ="a",
+        imageType ="DYNAMIC",
+    }
+}
+
+local upperBoundRequest = {
+    longitudeDegrees = 180.0,
+    latitudeDegrees = 90.0,
+    address = {
+        countryName = string.rep("a", 200),
+        countryCode = string.rep("a", 50),
+        postalCode = string.rep("a", 16),
+        administrativeArea = string.rep("a", 200),
+        subAdministrativeArea = string.rep("a", 200),
+        locality = string.rep("a", 200),
+        subLocality = string.rep("a", 200),
+        thoroughfare = string.rep("a", 200),
+        subThoroughfare = string.rep("a", 200)
+    },
+    timestamp = {
```

Comment:
@Itileda according to MOBILE_API this param should be `"timeStamp"`

Reply:
@AKalinich-Luxoft , fixed in a1dd9de8f0fb3a7642d9a9d9897aff95e763f59e